closest.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+export default function closest(options, goal) {
+  return options.reduce((prev, curr) => {
+    var val = Math.abs(curr - goal) < Math.abs(prev - goal) ? curr : prev;
+    return val > goal ? val : prev;
+  });
+}
```

Comment:
I was just copying the method over, but your refactor looks good. Keep in mind though that this is ES6 and not supported without a polyfill. Luckily, it looks like RN polyfills it https://github.com/facebook/react-native/blob/ed47efe4a17a6fa3f0a2a8a36600efdcd1c65b86/packager/react-packager/src/Resolver/polyfills/Array.prototype.es6.js